src/main/java/me/totalfreedom/totalfreedommod/command/Command_banip.java

```diff
@@ -75,6 +75,10 @@ else if (args.length > 1)
                 String message = ChatColor.RED + sender.getName() + " - Banned " + (plugin.sl.isStaff(player) ? "the IP " + ip : "an IP");
                 player.sendMessage(message);
             }
+            else
```

Comment:
Well, I added that because it should say "Banned [player] quietly." whenever the command is silent, so i just added that to the end of the if !silent as an else statement so that way, if it is silent, it will do the normal thing, and if not then it will send that message instead, I think the implementation is right, but i could be mistaken